IFC4x3/Properties/n/NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7" Name="NumberOfTreatmentHoles" UniqueId="e15c718c-2445-42cf-9506-a797f7772647" PrimaryDataType="IfcCountMeasure" />
```

Comment:
Propose renaming to "TreatmentHoleCount".

Reply:
Fixed in [fa34e3f](https://github.com/bSI-InfraRoom/IFC-Specification/pull/676/commits/fa34e3f56f97c85fda90a91839411efbd6712470)